op-chain-ops/genesis/layer_two.go

```diff
@@ -83,6 +83,7 @@ func BuildL2Genesis(config *DeployConfig, l1StartBlock *types.Block) (*core.Gene
 			if err != nil {
 				return nil, err
 			}
+			backend.Close()
```

Comment:
unlikely, but we probably also want to close it if the call above fails
```suggestion
			if err != nil {
				backend.Close()
				return nil, err
			}
			backend.Close()
```
or alternatively wrap that section in a `if err := func() err { backend, err := ...; defer backend.Close(); ...}(); err != nil { return nil, err}`